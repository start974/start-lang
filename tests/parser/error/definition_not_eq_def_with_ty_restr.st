def a : N = 32
