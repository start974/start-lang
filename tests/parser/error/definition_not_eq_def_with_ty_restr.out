tests/parser/error/definition_not_eq_def_with_ty_restr.st:1:11
1 | def a : N = 32
              ^
Error: Expected operator ':='.
